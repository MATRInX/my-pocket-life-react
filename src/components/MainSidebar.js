import React from 'react';
import AboutMe from './AboutMe';
import Archive from './Archive';
import FavouritePosts from './FavouritePosts';
import FacebookFanpage from './FacebookFanpage';
import InstagramFeed from './InstagramFeed';

const MainSidebar = () => (
  <aside>
    <AboutMe />
    <InstagramFeed />
    <FavouritePosts />
    <FacebookFanpage />
    <Archive />
  </aside>
);

export default MainSidebar;