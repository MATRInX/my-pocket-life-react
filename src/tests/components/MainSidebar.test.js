import React from 'react';
import { shallow } from 'enzyme';
import MainSidebar from '../../components/MainSidebar';

test('should correctly render MainSidebar', () => {
  const wrapper = shallow(<MainSidebar />);
  expect(wrapper).toMatchSnapshot();
});