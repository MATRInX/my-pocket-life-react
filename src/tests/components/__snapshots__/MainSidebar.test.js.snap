// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render MainSidebar 1`] = `
<aside>
  <Connect(AboutMe) />
  <Connect(InstagramFeed) />
  <Connect(FavouritePosts) />
  <FacebookFanpage />
  <Connect(Archive) />
</aside>
`;
